test/sql/filter/test_try_filter_doesnt_mutate_columns.test

```diff
@@ -0,0 +1,160 @@
+# name: test/sql/expressions/test_try_filter_doesnt_mutate_columns.test
+# description: Test to ensure TRY inside a filter does not mutate the output data
+# group: [filter]
+
+# Reproduction of the bug
+statement ok
+PRAGMA enable_verification;
```

Comment:
This is no longer needed - we have dedicated CI runs where verification is enabled on queries now.